stories/FileTreeNode.stories.tsx

```diff
@@ -82,8 +84,11 @@ storiesOf('FileTreeNode', module).addWithChapters('all variants', {
           sectionFn: () => {
             const versionId = 921;
             const store = createStoreWithVersion({
-              ...fakeVersion,
-              id: versionId,
+              version: {
+                ...fakeVersion,
+                id: versionId,
+              },
+              makeCurrent: true,
```

Comment:
Why does this have `makeCurrent` set to `true`?

Reply:
The old code was not doing that

Reply:
> The old code was not doing that

I must have missed it. 
Updated it now! :)